dataactvalidator/interfaces/interfaceHolder.py

```diff
@@ -23,31 +23,5 @@ def __init__(self):
 
     def close(self):
         """ Close all open connections """
-        InterfaceHolder.closeOne(self.jobDb)
-        InterfaceHolder.closeOne(self.errorDb)
-        InterfaceHolder.closeOne(self.stagingDb)
-        InterfaceHolder.closeOne(self.validationDb)
-
-    @staticmethod
-    def closeOne(interface):
-        """ Close all aspects of one interface """
-        if(interface == None):
-            # No need to close a nonexistent connection
-            return
-        try:
-            if(interface.session == None):
-                # If session is None, skip closing
-                return
-        except AttributeError as e:
-            # If interface has no session, skip closing
-            return
-
-        # Try to close the session and connection, on error try a rollback
-        try:
-            interface.session.close()
-        except:
-            try:
-                interface.session.rollback()
-                interface.session.close()
-            except:
-                pass
\ No newline at end of file
+        if self.jobDb is not None:
+            self.jobDb.close()
```

Comment:
Same question as above, for the broker interfaceHolder...this isn't covered by the cleanup code in app.py?